src/main/java/duke/Event.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+public class Event extends Task{
```

Comment:
Perhaps there could be a header comment here to help users understand what this code does?

```
/**
 * The Event class is a child class of Task.
 * Contains additional "date" string parameter.
 * @author  
 * @version 
 * @since  
 */
```

Similar comment for the other classes to ensure you don't lose marks unnecessarily due to CheckStyle issues :)

Reply:
Cool thanks bro.